install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -1132,6 +1132,7 @@ data:
   mesh-auth-mutual-enabled: "true"
   mesh-auth-mutual-listener-port: {{ .Values.authentication.mutual.port | quote }}
   mesh-auth-spire-agent-socket: {{ .Values.authentication.mutual.spire.agentSocketPath | quote }}
+  mesh-auth-mutual-connect-timeout: {{ .Values.authentication.mutual.connectTimeout | quote }}
```

nit: Consider using [`validateDuration`](https://github.com/cilium/cilium/blob/e8618e952098c9c05f2249afe80835d7d37291f7/install/kubernetes/cilium/templates/_helpers.tpl#L146-L156) For Golang duration formatted Helm values, e.g.

https://github.com/cilium/cilium/blob/e8618e952098c9c05f2249afe80835d7d37291f7/install/kubernetes/cilium/templates/cilium-configmap.yaml#L312-L316

```suggestion
  mesh-auth-mutual-connect-timeout: {{ include "validateDuration" .Values.authentication.mutual.connectTimeout | quote }}
```